docs/week3/03-architecture.html

```
@@ -0,0 +1,1008 @@
+        participant OR as Order<br/>Repository
+
+        Client->>Filter: POST /api/v1/orders
+        Filter->>Ctrl: userId = parseLong(헤더)
+        Ctrl->>Fac: place(request)
+        Fac->>Svc: place(userId, command)
+        Note over Svc: @Transactional 시작
+        Svc->>Svc: items 빈 체크 → 400
+
+        Svc->>DS: place(userId, items)
+        DS->>PR: findAllByIdIn(productIds)
+        PR-->>DS: List<Product>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)